### PR TITLE
First deployed version for parkertimmins

### PR DESCRIPTION
**Pull request by parkertimmins**
#### Check List:
- [x] Tests pass (`./test.sh <username>` shows no differences between expected and actual outputs)
- [x] All formatting changes by the build are committed
- [x] Your launch script is named `calculate_average_<username>.sh` (make sure to match casing of your GH user name) and is executable
- [x] Output matches that of `calculate_average_baseline.sh`
* Execution time: 29,015s
* Execution time of reference implementation: 4m44,720s
* System: Intel i7-5600U CPU @ 2.60GHz, 4 cores, 8GB RAM

Version notes:
        - simd search with overlapping words, at most one entry/semicolon parsed per simd word
        - branchless temperature parsing

### CODE REVIEW

**Comment by gunnarmorling** [comment]
00:07.913, nice!

File: calculate_average_parkertimmins.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+
+JAVA_OPTS="--enable-preview --add-modules jdk.incubator.vector"
+time java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_parkertimmins
```
Comment (gunnarmorling) [inline calculate_average_parkertimmins.sh:20]:
```suggestion
java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_parkertimmins
```

**Comment by ricardopieper** [comment]
Hey @gunnarmorling I think the leaderboard is a bit wrong, this entry is faster than mine, but I appear before it.

**Comment by gunnarmorling** [comment]
Whoops, good spot, thx! PR welcome to swap the order.

**Pull request merged**